css/index.css

```diff
@@ -0,0 +1,309 @@
+  margin-top: 40px;
+}
+
+.monthly {
+  font-size: 16px;
+  line-height: 24px;
+  color: #203a62;
+  margin-right: 12px;
+}
+
+.img-switch {
+  margin-right: 12px;
+}
+
+.annually {
+  font-size: 16px;
+  line-height: 24px;
+  color: #6f809a;
+  margin-right: 12px;
+}
+
+.save {
+  font-size: 14px;
+  line-height: 21px;
+  color: #74ca77;
+}
+
+.main {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 32px auto 0 auto;
+  column-gap: 20px;
+  row-gap: 50px;
+  flex-wrap: wrap;
+  padding-bottom: 80px;
+  box-sizing: border-box;
+}
+
+.plan {
+  width: 320px;
+  height: 593px;
+  background: #ffffff;
+  box-shadow: 0px 4px 6px rgba(12, 36, 71, 0.07);
+  border-radius: 17px;
+  padding: 32px;
+  box-sizing: border-box;
+  position: relative;
+}
+
+.img1 {
+  margin-right: 16px;
+}
+
+.block-header {
+  display: flex;
+}
+
+.header-text {
+  display: flex;
+  flex-direction: column;
+}
+.plan-name {
+  font-family: "Poppins";
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 32px;
+  margin-bottom: 9px;
+}
+
+.dollar {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 36px;
+  color: rgba(12, 36, 71, 0.3);
+}
+
+.number {
+  font-family: "Poppins";
+  font-size: 32px;
+  line-height: 48px;
+  font-weight: 700;
+  font-style: normal;
+}
+
+.month {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 36px;
+  color: rgba(12, 36, 71, 0.3);
+}
+.info-container {
+  margin-top: 24px;
+  display: flex;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.note {
+  margin-right: 8px;
+}
+.songs {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+  margin-right: 34px;
+}
+
+.songs-2 {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+  margin-right: 23px;
+}
+
+.storage {
+  margin-right: 8px;
+  align-items: center;
+}
+
+.memory {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.line {
+  width: 100%;
+  height: 1px;
+  background-color: #e4e8ec;
+}
+
+.list {
+  list-style: none;
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  margin-bottom: 16px;
+  margin-top: 24px;
+  color: rgba(12, 35, 70, 0.8);
+}
+
+.chek-box {
+  margin-right: 16px;
+}
+
+li {
+  margin-bottom: 16px;
+}
+
+.list-item {
+  display: flex;
+  align-items: center;
+}
+
+.btn-choose {
+  position: absolute;
+  bottom: 32px;
+  background-color: #3983f2;
+  border-radius: 10px;
+  border: none;
+  box-sizing: border-box;
+  padding: 18px 24px 14px 24px;
+  display: flex;
+  align-items: center;
+  box-shadow: 0 5px 0 #3c354a;
+  transition-duration: 1s, 100ms, 100ms;
+  transition-property: background-color, bottom, box-shadow;
+  transition-delay: 0s, 0.5s, 0.5s;
```

Comment:
https://developer.mozilla.org/ru/docs/Web/CSS/transition read it plz

Reply:
Can u describe more  informative what you mean ?